Extensions/PanelSpriteObject/panelspriteruntimeobject-pixi-renderer.ts

```diff
@@ -25,6 +26,7 @@ namespace gdjs {
         );
```

Comment:
You can take the opportunity to remove the useless `if (this._spritesContainer === undefined) {` .
This is dating from the moment where we were calling constructor on already existing objects to "recycle" objects. We then switched to a proper separate method called `reinitialize` (which is not implemented by most objects, but could be to reduce the memory pressure if a lot of objects are created/removed and we identify that some objects are costly to instanciate).